src/router/AppRouter.tsx

```diff
@@ -4,7 +4,7 @@ import {APIPage} from "../pages/APIPage";
 
 export const router = createBrowserRouter([
     {
-        path: '/apidocs',
+        path: process.env.PUBLIC_URL,
```

Comment:
`/` is fine here.

We need to configure below the basehref  like this:
```
export const router = createBrowserRouter([
    {
        path: '/',
        element: <LandingPage />
    },
    {
        path: '/api/:api',
        element: <APIPage />
    }
], {
    basename: process.env.PUBLIC_URL
});
```

Reply:
Basename will configure the base of our app, so that everything else can safely relay on the base without needing to be explicit.